src/components/navigation/nav-items/tenant-selector.scss

```diff
@@ -0,0 +1,25 @@
+.tenant-dropdown .dropdown-toggle::after {
+  float: right;
+  margin-top: 0.75em;
+}
+
+.tenant-dropdown.nav-item.dropdown {
+  border: 1px solid white;
+  border-radius: 5px;
+  margin-top: 5px;
+  width: 93%;
+}
+
+.tenant-dropdown .dropdown-toggle.nav-link {
+  padding-top: 0.5em;
+  padding-bottom: 0.5em;
+}
+
+.tenant-selector-one-tenant {
+  margin-top: 0.5em;
+  margin-bottom: 0.5em;
+}
+
+.tenant-dropdown .dropdown-menu.show {
+  width: 100%;
```

Comment:
Bootstap have the class `w-100` that can do the same.

Reply:
I do not have access to the DOM of this, since it comes from NavDropdown